assets/stylesheets/_components.scss

```diff
@@ -284,28 +284,6 @@
 @import 'my-sites/post-type-list/post-type-site-info/style';
 @import 'my-sites/posts/style';
 @import 'my-sites/plugins/style';
```

Comment:
Can you also migrate the `my-sites/plugins/style` stylesheet? It should be included from `plugins/main.jsx`.

It also contains an unused `.card.is-compact.section-header.after-compact` rule that can be removed.

Thanks for working on the migration!

Reply:
:+1:

Thanks for confirming!